README.md

```diff
@@ -130,6 +130,10 @@ The exact viewport of the calendar. When these are specified, scrolling in the c
 
 **Note that you need to provide either `defaultTimeStart/End` or `visibleTimeStart/End` for the timeline to function**
 
+## utcOffset
```

Comment:
If we apply a `utcOffset` to the Timeline how does this PR handle timeline events such as `onItemMove` and `onItemResize`?

Will these callbacks send back UTC timestamps or timestamps with the UTC offset still applied?

Reply:
I didn't use it since this PR, so what I will say need to be verified : if you give an utcOffset, all computation are relative to the offset, so normally, you will receive in callback, times with the offset applied.

Reply:
I'm thinking that we might want to convert it back to UTC in the callback. The reason is the following scenario:

- Database stores timestamps in UTC
- API sends back data in UTC
- Timeline uses UTC offset to show these items in the user's timezone
- When a user updates the time of the item, the callback is used to send back the timestamp in UTC so that the API can store it in the DB

In this way, all data that goes **into** the timeline is UTC and all data that goes **out** of the timeline is in UTC. Or if you're not using an `utcOffset` all data going into the `timeline` comes out in the same way.

What do you think?

Reply:
I think `react-calendar-timeline` doesn't have to take care of that, it outputs what you asked him, and you still able to convert the time in utc in callbacks 😉

Reply:
Hmm, but then you could argue that you might as well just convert your input `items[n].start_time` and `items[n].end_time`, `canvasTimeStart`, etc. to the UTC offset that you need, right?

@Ilaiwi curious to hear your thoughts as I'm pretty new to the codebase of `react-calendar-timeline`

Reply:
I think especially that `react-calendar-timeline` is not a time converter, instead of `moment`